{"name": "困人处置列表-真实", "status": "passed", "attachments": [{"name": "log", "source": "ca0d797f-0316-442d-8998-bc1d316f3285-attachment.txt", "type": "text/plain"}], "start": 1692689153524, "stop": 1692689153761, "uuid": "da6c11ee-4d4e-4450-b4ba-67736313c0f8", "historyId": "2c9991b785716977eb4cbbaa1f323277", "testCaseId": "792deb5b60e48cc51a654ad1e9657ccb", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_003", "labels": [{"name": "story", "value": "应急处置"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}